### ISSUE TITLE
ZClassic Electrum 1.0.6 Server Lagging / Not Connecting

### ISSUE THREAD

**Issue by Achieveaholics** [BTCP-community/electrum-zcl#194]
Wondering if anyone else can't connect to the ZCL server? Is there another server the wallet is able to connect to, currently it is stuck on lagging. Does this usually resolve on itself or what are the next steps to do next? 

Thank you for your time!

**Comment by NATURALCOIN** [comment]
I had the same problem today when I opened my wallet. I hit up the discord and found the server below mentioned in the help channel. Added it to the server list and my wallet connected fine.

electrum.zclcore.com:50002

**Comment by ghost** [comment]
@NATURALCOIN 🎉

I had the same issue as @Achieveaholics.

I find nothing but electrum.zclassic.community:50002 on Internet, and it's dead or something.

electrum.zclcore.com:50002 works like a charm !

Thanks !

**Comment by mexican1973** [comment]
you can also please add the second server which is also up and in synch 24/7 👍 
electrum.schlachter.mx:50002 I added a pull / merge request but have no authorization.
Thats the one I am running. Cheers, Heinrich.

**Comment by ghost** [comment]
Thanks for the servers and ports !
I use yours @mexican1973.
Any news @Achieveaholics ?

**Comment by Achieveaholics** [comment]
@St3x I'm still currently using the electrum.zclcore.com:50002  server and it works just fine! Sent over a amount of funds, did not take long at all to update. Still up and running smooth!

**Comment by curlytastick** [comment]
I'm having the same issue so thanks for the server. Do you know how long it takes to sync? I remember it took quite a while before, but that was over a year ago!?

**Comment by Achieveaholics** [comment]
@curlytastick What do you mean by sync? (Like the time is takes funds being transferred over to another wallet address? Or connection towards the wallet server to gain access to send and receive?)

**Comment by mexican1973** [comment]
He means to download all the blocks. Depending on your internet connection it can take up to 12 hours to download the blockchain.

**Comment by curlytastick** [comment]
its telling me I am connected to 5 nodes with 1 block. its done nothing that I can tell - I guess its finished or does it take this long? I've been trying all day!

**Comment by mexican1973** [comment]
if you are trying the electrum walllet. Try to delete the servers you have form the peers.dat and add the two working servers.
electrum.schlachter.mx and electrum.zclcore.com
You will see in the overview screen you block height and the one form the servers.

**Comment by Achieveaholics** [comment]
The Electrum wallet server is not connecting again, is it down or is there another server I can use again? Thank you!

**Comment by Achieveaholics** [comment]
@NATURALCOIN The Electrum wallet server is not connecting again, is it down or is there another server I can use again? Thank you!

**Comment by NATURALCOIN** [comment]
I'm having the same problem. None of the servers I have are working now. Not sure why. I actually downloaded the QT ZCL wallet since I'm running Windows 7 and it connects just fine. I had to manually download the params though to get it working at first. I now need to import my keys from my Electrum wallet into it so my ZCL shows up.

Another alternative is the Zelcore wallet from the Zelcash team (https://zeltrez.io/#). I've been using Discord to communicate with the ZCL team and community (https://discord.gg/vgErStB).

**Comment by mexican1973** [comment]
There is an issue with the electrumx server after the sapling upgrade, we are working  on the fix as of today and hope to upgrade server by latest tomorrow.

**Comment by NATURALCOIN** [comment]
Once the issue is fixed which server or servers should be used to connect?

**Comment by mexican1973** [comment]
I can speak only for the server I am running:
electrum.zclassic.ch  port 50002
The second server which also should be working is:
electrum.zclcore.com port 50002

**Comment by mexican1973** [comment]
electrum.zclassic.ch is same as electrum.schlachter.mx  
It seemed more reasonable to use a zclassic domain than my private domain.

**Comment by NATURALCOIN** [comment]
Thanks and your server is running normally?

**Comment by curlytastick** [comment]
Cheers Mexican, but every time I try to connect it won’t sync? I’ve left it syncing for a couple of hours too...

**Comment by mexican1973** [comment]
yes it runs completely smooth. Just now a bug with sapling. Will catch up soon and update here.

**Comment by mexican1973** [comment]
At the moment you cannot access via electrum client and ledger device because a bug in electrum client software due to sapling upgrade. My colleague is working on it.
Please follow the chat in Zclassic2 or zclassic for any update. Should be working in the next days again.

**Comment by NATURALCOIN** [comment]
Thanks @mexican1973 ! Greatly appreciated. Are you referring to the chat in Discord zclassic2?

**Comment by mexican1973** [comment]
No on telegram I meant:
https://t.me/ZclassicCE
https://t.me/ZClassicCoin

**Comment by NATURALCOIN** [comment]
Thanks! i joined both.

**Comment by davidreverett** [comment]
Any updates? I'm having the same issues.
